src/test/java/net/openhft/chronicle/wire/ReadmeChapter1Test.java

```diff
@@ -181,6 +182,13 @@ public void example2() {
         bytes2.releaseLast();
     }
 
+    @Test
+    public void abstr() {
```

Comment:
This test should have checks maybe?

Reply:
Ahh. This test should be removed. I will do that in a new commit